group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -8148,7 +8174,10 @@ private TaskAssignor streamsGroupAssignor(String groupId) {
      * Get the assignor of the provided streams group.
      */
     private Map<String, String> streamsGroupAssignmentConfigs(String groupId) {
-        return Map.of("group.streams.num.standby.replicas", "0");
+        Optional<GroupConfig> groupConfig = groupConfigManager.groupConfig(groupId);
+        final Integer numStandbyReplicas = groupConfig.map(GroupConfig::streamsNumStandbyReplicas)
+            .orElse(config.streamsGroupNumStandbyReplicas());
+        return Map.of("num.standby.replicas", numStandbyReplicas.toString());
```

Comment:
Why we we return without `group.streams` prefix of the config name now?

Reply:
This is just convention. We don't need the prefix in this map. But we could keep it.

Reply:
Ok, was just wondering because the existing code does use the prefix.